app/src/main/java/com/github/gotify/messages/ListMessageAdapter.java

```diff
@@ -65,7 +71,7 @@ public View getView(int position, View convertView, ViewGroup parent) {
         final MessageWithImage message = items.get(position);
         holder.message.setText(message.message.getMessage());
         holder.title.setText(message.message.getTitle());
-        picasso.load(message.image)
+        picasso.load(imageBaseUrl + "/" + message.image)
```

Comment:
It would be great if the app still supports old versions of gotify/server. We could either check if the image url already is absolute (with host) or check the version of gotify/server and then decide how the image url should be handled.